packages/graphql-language-service-interface/src/getDiagnostics.ts

```diff
@@ -31,10 +31,21 @@ import {
 import { DiagnosticSeverity, Diagnostic } from 'vscode-languageserver-types';
 
 export const SEVERITY = {
-  ERROR: 1 as DiagnosticSeverity,
-  WARNING: 2 as DiagnosticSeverity,
-  INFORMATION: 3 as DiagnosticSeverity,
-  HINT: 4 as DiagnosticSeverity,
+  Error: 'Error' as 'Error',
```

Comment:
@rebornix i think this was even more along the lines of what you were thinking of?